examples/api/date_index_formatter.py

```diff
@@ -1,4 +1,8 @@
 """
+====================================
+Custom tick formatter for timeseries
+====================================
+
 When plotting time series, e.g., financial time series, one often wants
 to leave out days on which there is no data, eh weekends.  The example
```

Comment:
The existing "eh" here is probably not someone being Canadian... Probably should be "e.g.," or maybe "i.e.,".